libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -1062,7 +1062,7 @@ method rpcHandler*(g: GossipSub,
     # g.anonymize needs no evaluation when receiving messages
     # as we have a "lax" policy and allow signed messages
 
-    if not (await g.validate(msg)):
+    if g.validate(msg).await == ValidationResult.Reject:
```

Comment:
Can we just do `if (await g.validate(msg)) == ValidationResult.Reject:` - seems a lot more natural and consistent.

Reply:
hehe ok, I really hate .net and c# to start the `await\s.*` trend to be honest.. yes it became natural but in a weird way to be honest.